cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1011,12 +1011,12 @@
 
 ;; val is_signed from_bits to_bits
 (decl lower_extend(Reg bool u8 u8)ValueRegs)
```

Comment:
likewise here -- pre-existing but would you mind doing a search for `[A-Za-z0-9](` and `)[A-Za-z0-9]` and fixing spaces around paren-forms?

(happy to do this in a separate PR if you'd rather get this in as-is but if it's easy...)